templates/extensions/the-graph/README.md.args.mjs

```diff
@@ -0,0 +1,303 @@
+export const extraContents = `## 🚀 Setup The Graph Integration
+
+Now that we have spun up our blockchain, started our frontend application and deployed our smart contract, we can start setting up our subgraph and utilize The Graph!
+
+> Before following these steps be sure Docker is running!
+
+&nbsp;
```

Comment:
Feels weird to have that if reading the source.

Reply:
Lol yup removed `&nbsp;`  them at [b4336a4](https://github.com/scaffold-eth/create-eth/pull/25/commits/b4336a44b28da244fba45fbe78dbf9f94121917b) since they weren't adding too much value visually too when you render markdown